packages/core/src/Swatch/Swatch.tsx

```diff
@@ -0,0 +1,36 @@
+import PropTypes, { InferProps } from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+import { Flex } from '../Flex'
+
+const SwatchColor = styled.div`
+  height: 24px;
+  width: 24px;
+  background-color: ${(props) => props.color};
+  border-radius: 50%;
+  display: flex;
+  margin: 4px;
+`
```

Comment:
could you add `cursor: pointer;` to indicate that it's clickable

Reply:
Thanks, that's a good catch! I changed it if onClick is provided then we show a pointer cursor otherwise we show the regular cursor.

Reply:
`mouse` isn't a valid value, you want `default`

Reply:
Right.. 😅 changed to default